soc/arm/renesas_smartbond/da1469x/soc.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SOC__H_
+#define _SOC__H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+
+#include <devicetree.h>
+#include <sys/util.h>
+#include "DA1469xAB.h"
```

Comment:
these are likely not needed, please remove. `soc.h` needs to exist on ARM because certain MPU code relies on it for indirect dependencies (needs to be fixed). Just include the bare minimum for that case.